inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/embedding_segments_sum.cpp

```diff
@@ -11,18 +12,43 @@ using namespace LayerTestsDefinitions;
 
 namespace {
 
-const std::vector<InferenceEngine::Precision> netPrecisions = {
-        InferenceEngine::Precision::FP32,
-        InferenceEngine::Precision::I32,
-        InferenceEngine::Precision::U8
+const std::vector<ov::test::ElementType> netPrecisions = {
+        ov::test::ElementType::f32,
+        ov::test::ElementType::i32,
+        ov::test::ElementType::u8
 };
 
-const std::vector<InferenceEngine::Precision> indPrecisions = {
-        InferenceEngine::Precision::I64,
-        InferenceEngine::Precision::I32
+const std::vector<ov::test::ElementType> indPrecisions = {
+        ov::test::ElementType::i64,
+        ov::test::ElementType::i32
+};
+
+const std::vector<ov::test::InputShape> input_shapes = {
+    // dynamic input shapes
+    {
+        // input model dynamic shapes
+        {ov::Dimension::dynamic(), ov::Dimension::dynamic()},
+        // input tensor shapes
+        {{5, 6}, {10, 35}}
+    },
+    {
+        // input model dynamic shapes
+        {ov::Dimension::dynamic(), ov::Dimension::dynamic(), ov::Dimension::dynamic()},
```

Comment:
Just minor comment. You will validate just dynamic case not static

Reply:
discussde offflline: we are testing dynamic cases and static cases.